### ISSUE TITLE
No processing of toml if oracledb_up is 0

### ISSUE THREAD

**Issue by eagle-rr** [iamseth/oracledb_exporter#220]
Is there a parameter which can be configured whereby you tell oracledb_exporter to not process the sql in the toml file(s) if oracledb_up is 0 value?  If oracledb_exporter could just provide output of oracldb_up as 0 immediately (instead of trying to process the sql in toml files); this would allow us to more quickly receive up/down metric/alerts (instead of each sql having to timeout first).

**Comment by Yannig** [comment]
No, I think it's not supported. Pull request are welcome though.
